replication-entity/replication-entity-controller/src/main/resources/templates/replication/admin/page/controller.vm

```diff
@@ -99,6 +100,22 @@
   </select>
 #end
 
+#macro(selectDirection $currentDirection, $inputId, $showDirectionSelect)
+  #set ($cssClass = '')
+  #if (!$showDirectionSelect)
+    #set ($cssClass = 'hidden')
+  #end
+  <label class="direction-label $escapetool.xml($cssClass)">$escapetool.xml($services.localization.render('replication.entity.direction'))</label>
```

Comment:
It's not clear to me why you feel the need to escape `$cssClass` when it's can only be "hidden" or empty.

Reply:
Yes, it's not needed in this case ... it's a best practices reflex :D